libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/JObjectConverter.cs

```diff
@@ -2,21 +2,20 @@
 // Licensed under the MIT License.
 
 using System;
-using Microsoft.Bot.Builder.Dialogs.Adaptive;
 using Microsoft.Bot.Builder.Dialogs.Debugging;
-using Microsoft.Bot.Builder.Dialogs.Declarative.Resolvers;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Bot.Builder.Dialogs.Declarative.Converters
 {
     public class JObjectConverter : JsonConverter
     {
```

Comment:
Documentation is missing for this whole class.

Reply:
only for the class itself, the methods are overrides.

---
In reply to: [376541996](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376541996) [](ancestors = 376541996)